schematics/cloud/authentication/emails/files/emailsLambda.ts.template

```diff
@@ -79,18 +84,24 @@ export const handler: CustomMessageTriggerHandler = async (
 
       const resetLink = generateCodeLink(triggerSource, email);
 
+      const reformedReplacements: [RegExp, string][] = [
+        [/%RESETPASSWORD%/g, resetLink],
+      ];
+
       event.response.emailSubject = "<%= classify(name) %> | Reset password";
-      event.response.emailMessage = await generateEmailFromS3Template(
-        s3,
-        EmailTemplate.RESET_PASSWORD,
-        [[/%RESETPASSWORD%/g, resetLink]]
-      );
+      event.response.emailMessage = await generateEmailFromS3Template(s3, {
+        templateName: EmailTemplate.RESET_PASSWORD,
+        bucketName: process.env.TEMPLATE_BUCKET_NAME!,
+        reformedReplacements,
+      });
     }
   } catch (e) {
-    console.log(
-      `Error occurred while fetching event ${event.triggerSource}. I am going to send default Email message!`
-    );
-    console.error(e);
+    handleError(e);
+    handleError(
```

Comment:
Can we show original error message? I'm guessing that it can make debugging process a bit easier.

Is `necessary` required here?

Reply:
actually we can inside `handleError`

Reply:
in case of error lambda displays errors out of the box